tap_hubspot/client.py

```diff
@@ -155,6 +129,14 @@ def request(self,
             metrics_status_code = response.status_code
 
             timer.tags[metrics.Tag.http_status_code] = metrics_status_code
+        
+        if response.status_code == 403:
+            LOGGER.warn('Invalid Auth Exception - 403')
+            raise InvalidAuthException(response.text)
```

Comment:
I don't think it's worth retrying here when it gets a 403, so I'd expect it to stop. The log line however is redundant because when the exception is raised, we already know where we are.

Reply:
Why is the logging redundant? Wouldn't it make it easier to query in splunk with the string 'Invalid Auth Exception -403'